packages/xod-client-browser/src/containers/App.jsx

```diff
@@ -61,6 +61,10 @@ class App extends client.App {
     props.actions.openProject(props.tutorialProject);
   }
 
+  _onRunSimulation() {
+    this.onRunSimulation(true);
```

Comment:
Explicitly telling "hey, remember, we are in a browser" each time the method is called seems a bit strange. It's not like it would change during program execution.